src/rvpt/rvpt.cpp

```diff
@@ -182,10 +183,24 @@ void RVPT::update_imgui()
         ImGui::PushItemWidth(80);
         ImGui::SliderInt("AA", &render_settings.aa, 1, 64);
         ImGui::SliderInt("Max Bounce", &render_settings.max_bounces, 1, 64);
-        if (ImGui::Button("Debug Raster")) toggle_debug();
-        ImGui::SameLine();
-        if (ImGui::Button("Wireframe")) toggle_wireframe_debug();
 
+        ImGui::Checkbox("Debug Raster", &debug_overlay_enabled);
```

Comment:
Since you moved the wireframe checkbox to the next line, you lengthen the text without issue.
"Debug Raster Overlay" or "Raster Debug View" or something more descriptive.